packages/cozy-client/src/types.js

```diff
@@ -92,12 +92,30 @@ import { QueryDefinition } from './queries/dsl'
  */
 
 /**
- * A reference to a document (special case of a relationship used between photos and albums)
+ * @typedef {object} Relationships
```

Comment:
I don't think we should name it `Relationships`. You already defined a `Relationship` here https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/associations/HasMany.js#L9

I think we should name it `ReferencedByRelationship` for instance.